test/integration/fixtures/VerticalLinearStepper.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
```

Comment:
What do you think about?
```js
import React, {Component, PropTypes} from 'react';
```
I hope we would be able to get rid of this global React import in the future.